src/App.re

```diff
@@ -0,0 +1,45 @@
+open Chrome.Extensions;
+open Chrome.Runtime;
+
+type route = 
+  | Login
+  | JobApp
+  | NotFound;
+
+type state = {route};
+
+type action =
+  | Login
+  | Logout;
+
+let reducer = (action, state) =>
+  switch (action) {
+  | Login => ReasonReact.Update({route: JobApp})
+  | Logout => ReasonReact.Update({route: Login})
+  };
+
+let component = ReasonReact.reducerComponent("App");
+
+let make = _children => {
+  ...component,
+  reducer,
+  initialState: () => {route: Login},
+  render: self =>
+  <div>
+    (switch (self.state.route) {
+    | Login => <LoginPage />
+    | JobApp => <JobAppPage />
```

Comment:
For me to see changes made in the `LoginPage` & `JobAppPage` components, I had to rename the files to `LoginPage.re` & `JobAppPage.re`.
IIRC Reason didn't need explicit imports because it will load the files with the same names as components.

Question: with the current naming of the components, are you able to see changes done to `Login` & `JobApp`?
if so, then its a bug on my end and doesn't need to be addressed in the PR

Reply:
you're not on latest commit I think (?), I changed these